components/infobox/wikis/counterstrike/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,249 @@
+---
+-- @Liquipedia
+-- wiki=counterstrike
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Namespace = require('Module:Namespace')
+local Page = require('Module:Page')
+local Player = require('Module:Infobox/Person')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+
+local _BANNED = mw.loadData('Module:Banned')
+local _ROLES = {
+	-- Players
+	['awper'] = {category = 'AWPers', display = 'AWPer'},
+	['igl'] = {category = 'In-game leaders', display = 'In-game leader'},
+	['lurker'] = {category = 'Riflers', display = 'Rifler', category2 = 'Lurkers', display2 = 'lurker'},
```

Comment:
this plus the lines below use rifler stuff for cat, display
is that intended?
(might be correct just seems odd to me)

Reply:
I _think_ that Lurker etc are subroles of Riflers

Reply:
okay